etl/scripts-ccao-data-warehouse-us-east-1/spatial/spatial-parcel.R

```diff
@@ -173,28 +226,159 @@ process_parcel_file <- function(s3_bucket_uri,
         filter(!is.na(lon) & !is.na(x_3435)) %>%
         bind_rows(spatial_df_missing)
     }
+    tictoc::toc()
+
+    tictoc::tic(paste("Calculated shape features:", file_year))
+    # Simplify the planar geometry to reduce noise in shape-based features
+    # calculated below. Convert to a data.table of the vertices of each polygon.
+    # Using data.table here because it's much faster than dplyr
+    spatial_mat_coords <- spatial_df_merged %>%
+      st_set_geometry("geometry_3435") %>%
+      st_simplify(dTolerance = 2, preserveTopology = TRUE) %>%
+      st_cast("MULTIPOLYGON") %>%
```

Comment:
Added a simplification step with a low, low tolerance (2 feet) to remove points that are basically overlapping.